lib/oid.js

```diff
@@ -25,8 +25,8 @@ function currentHour() {
 
 function oidFromDateTime(timestamp) {
   const ts = (typeof timestamp === 'string') ? new Date(timestamp) : (timestamp ?? currentHour());
-  const hexSeconds = Math.floor(ts / 1000).toString(16);
-  return ObjectId(hexSeconds + '0000000000000000'); // eslint-disable-line prefer-template
+  const hexSeconds = Math.floor(ts / 1000).toString(16).padEnd(24, '0');
```

Comment:
Yeah, this actually didn't work for dates back in the 70s and 80s: had one two few digits.   Not really an issue, but encountered it with unit tests so let's address it.